tests/e2e/utils/src/components.js

```diff
@@ -401,18 +401,19 @@ const addProductToOrder = async ( orderId, productName ) => {
 /**
  * Creates a basic coupon with the provided coupon amount. Returns the coupon code.
  *
+ * @param discountType Type of a coupon. Defaults to Fixed cart discount.
```

Comment:
When adding a new parameter to a function, the new parameter needs to be added after existing parameters. Changing parameter order breaks existing code using the function.

Reply:
Agree definitely, it's done. Thanks!